auth_supplier/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=============
+Auth Supplier
+=============
+
+This module was written to extends the functionality of auth signup
+and allows the user to create an account as a supplier.
+
+Configuration
+=============
+
+* To enable users to create accounts:
```

Comment:
You can include here that "Developer mode" should be activated first to have access to "Technical features".